Jenkinsfile

```diff
@@ -129,8 +129,9 @@ def checkAndAbortBuild() {
 }
 
 def startBraveBrowserBuild() {
+    PIPELINE_NAME = "pr-brave-browser-" + BRAVE_CORE_BRANCH.replace('/', '-')
```

Comment:
@mihaiplesa LGTM however we may want to consider escaping any other characters which may cause issues with the pipeline name at some point

Reply:
shouldn't be needed as in the past we only had to escape `/`